test/e2e/operator_test.go

```diff
@@ -430,6 +467,27 @@ func ensureOperandRoleBinding() error {
 	return kubeClient.Create(context.TODO(), &rb)
 }
 
+func ensureOperandClusterRoleBinding() error {
+	crb := rbacv1.ClusterRoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: operandRbacRsrcName,
+		},
+		RoleRef: rbacv1.RoleRef{
+			APIGroup: rbacv1.GroupName,
+			Kind:     "ClusterRole",
+			Name:     operandRbacRsrcName,
+		},
+		Subjects: []rbacv1.Subject{
+			{
+				Kind:      rbacv1.GroupKind,
```

Comment:
Can't we use kind: ServiceAccount  rather than using Group ?

https://kubernetes.io/docs/reference/access-authn-authz/rbac/#role-binding-examples

Reply:
Basically using this groups allows us to cover all existing and future serviceaccounts in external-dns namespace, without having to guess their names in advance. 
The alternative, using `kind: ServiceAccount` would force us to update the list of subjects in the clusterrolebinding, on every event occurring on the externalDNS instances

Reply:
Hmm, OK so basically you want all service accounts in external-dns ns to get that cluster role? Does it sound right ?

Reply:
Yes, that is what we agreed on yesterday.